src/components/Editors/Chart/Sections/Transform.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from 'react'
```

Comment:
It's better to put this file inside the Transform directory making it a complete "Transform" module